cloudsigma2-mia/pom.xml

```diff
@@ -0,0 +1,163 @@
+    </dependency>
+    <dependency>
+      <groupId>org.apache.jclouds.labs</groupId>
+      <artifactId>cloudsigma2</artifactId>
+      <version>${project.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.jclouds</groupId>
+      <artifactId>jclouds-core</artifactId>
+      <version>${project.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.jclouds</groupId>
+      <artifactId>jclouds-compute</artifactId>
+      <version>${project.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.jclouds.driver</groupId>
+      <artifactId>jclouds-log4j</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.jclouds.driver</groupId>
+      <artifactId>jclouds-sshj</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.auto.service</groupId>
+      <artifactId>auto-service</artifactId>
+      <optional>true</optional>
+    </dependency>
+  </dependencies>
+
+  <profiles>
+    <profile>
+      <id>live</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>integration</id>
+                <phase>integration-test</phase>
+                <goals>
+                  <goal>test</goal>
+                </goals>
+                <configuration>
+                  <systemPropertyVariables>
+                    <test.cloudsigma2-mia.endpoint>${test.cloudsigma2-mia.endpoint}</test.cloudsigma2-mia.endpoint>
+                    <test.cloudsigma2-mia.api-version>${test.cloudsigma2-mia.api-version}</test.cloudsigma2-mia.api-version>
+                    <test.cloudsigma2-mia.build-version>${test.cloudsigma2-mia.build-version}</test.cloudsigma2-mia.build-version>
+                    <test.cloudsigma2-mia.identity>${test.cloudsigma2-mia.identity}</test.cloudsigma2-mia.identity>
+                    <test.cloudsigma2-mia.credential>${test.cloudsigma2-mia.credential}</test.cloudsigma2-mia.credential>
+                    <test.cloudsigma2-mia.template>${test.cloudsigma2-mia.template}</test.cloudsigma2-mia.template>
+                  </systemPropertyVariables>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+  </profiles>
+
+  <build>
+  	<pluginManagement>
+  		<plugins>
+  			<!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself.-->
```

Comment:
IIRC, this is something that can be avoided by fixing it in eclipse? The pom issues I have can be fixed by right clicking the error > Quick Fix > and picking the one marked as (Experimental), which works well.